### ISSUE TITLE
Error when more than 3 mediators in serial model

### ISSUE THREAD

**Issue by Fdoel** [aalfons/robmedExtra#31]
Currently the app GUI allows users to input more than 3 mediators, but more than 3 mediators are not supported by robmed. When you try to run ROBMED with this model, the app just crashes. 

Would be good to prevent the user from inputting more than 3 mediators, and to display a message explaining why when 3 are selected.

**Comment by Fdoel** [comment]
Or rather than that, prevent the user from using the serial model when more than 3 mediators are selected, as preventing from inputting  more than 3 mediators would limit the parallel model.

**Comment by aalfons** [comment]
Good catch. Indeed, this kind of exception handling is still missing from the GUI. This also applies to other inputs, for example when a confidence level <0 or > 1 is entered, `robmed` gives a warning message on the console but then uses the default value, but there should be something that prevents that in the GUI.

There are a number of other issues with these kinds of behavior, but we still have to think about how we want to implement this.

**Comment by AuroreAA** [comment]
`validate` and `need` should be the way to deal with that here

**Comment by aalfons** [comment]
For the problem with serial mediators, I ended up with a different solution to prevent an error. When the user selects more than three mediators, the input for the type of mediation model disappears, and instead a note or warning is displayed that only the parallel mediator model is implemented for this case. Since we also show a diagram of the selected model now, there should not be any confusion over what the software does.

I will therefore close this issue with the relevant commit, but I'll open a more general issue for other inputs in subsequent tabs.
